src/app/common/pipes/utc-date-pipe/utc-date-utils.service.ts

```diff
@@ -8,13 +9,12 @@ export class UtcDateUtils {
 			let momentDate;
 			if (isMoment(value)) {
 				momentDate = value;
```

Comment:
@jrassa - are these changes simply not synced from the latest merge? I didn't expect to see `moment` here still.

Reply:
Correct.  I will get this synced shortly.